qmt/tasks/sweep.py

```diff
@@ -1,7 +1,7 @@
 import dask
 import dask.distributed
 from six import iteritems
-
+import cloudpickle
```

Comment:
You'll need to add `cloudpickle` to https://github.com/Microsoft/qmt/blob/master/requirements.txt.

Reply:
Good point, thanks!